<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="bClose">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>630</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Calibration</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>271</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QPushButton" name="bJoyButton4">
         <property name="text">
          <string>Joy B4</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="bJoyButton2">
         <property name="text">
          <string>Joy B2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="bJoyButton1">
         <property name="text">
          <string>Joy B1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="bJoyButton3">
         <property name="text">
          <string>Joy B3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>254</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>POV</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="bJoyPOVUp">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="bJoyPOVLeft">
         <property name="text">
          <string>Left</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="bJoyPOVDown">
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="bJoyPOVRight">
         <property name="text">
          <string>Right</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>594</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="3" column="1">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="bYAxisCal">
           <property name="text">
            <string>Calibrate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bYAxisCalView">
           <property name="text">
            <string>View</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="4">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="bZAxisCal">
           <property name="text">
            <string>Calibrate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bZAxisCalView">
           <property name="text">
            <string>View</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="2">
        <widget class="QSlider" name="YAxis">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32767</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QCheckBox" name="chbXAxisInvert">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Invert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lXAxisValue">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QCheckBox" name="chbYAxisInvert">
           <property name="text">
            <string>Invert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lYAxisValue">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>X-Axis</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QSlider" name="ZAxis">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32767</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="bXAxisCal">
           <property name="text">
            <string>Calibrate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bXAxisCalView">
           <property name="text">
            <string>View</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QSlider" name="XAxis">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32767</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QCheckBox" name="chbZAxisInvert">
           <property name="text">
            <string>Invert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lZAxisValue">
           <property name="text">
            <string>Z:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Y-Axis</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Z-Axis</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupCalibrate">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>210</y>
        <width>241</width>
        <height>361</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>231</width>
         <height>341</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="lCalibrateOption">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QDial" name="dCalibrateDeadband">
          <property name="maximum">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lCalibrateDeadband">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QDial" name="dCalibrateMin">
          <property name="minimum">
           <number>-32768</number>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="value">
           <number>-32768</number>
          </property>
          <property name="sliderPosition">
           <number>-32768</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lCalibrateMin">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QDial" name="dCalibrateMax">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>32768</number>
          </property>
          <property name="value">
           <number>32768</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="lCalibrateMax">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lCalibrateStep">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="chbInvert">
          <property name="text">
           <string>Invert</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="bCalibrateSave">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="bCalibrateCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <zorder>formLayoutWidget</zorder>
     <zorder>gridLayoutWidget</zorder>
     <zorder>gridLayoutWidget_2</zorder>
     <zorder>label_5</zorder>
     <zorder>gridLayoutWidget_4</zorder>
     <zorder>checkBox_2</zorder>
     <zorder>groupCalibrate</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
